src/families/location_scale.jl

```diff
@@ -57,17 +67,17 @@ Base.eltype(::Type{<:MvLocationScale{S, D, L}}) where {S, D, L} = eltype(D)
 function StatsBase.entropy(q::MvLocationScale)
     @unpack  location, scale, dist = q
     n_dims = length(location)
-    n_dims*convert(eltype(location), entropy(dist)) + first(logdet(scale))
+    n_dims*convert(eltype(location), entropy(dist)) + logdet(scale)
```

Comment:
Is this convert really needed?

It seems strange that `eltype(location) and `entropy(dist)` should result in different types?

Reply:
Nice catch. It's probably best to assume the user is responsible for setting the types correctly at this point. Let me fix this outside of this PR later